.github/workflows/ci.yml

```diff
@@ -18,11 +18,11 @@ jobs:
         run: |
           node scripts\download-hydra.js
           Get-ChildItem
-          Expand-Archive -Force -Path "cardano-wallet-jormungandr-*-win64.zip" -DestinationPath .
-          Expand-Archive -Force -Path "cardano-wallet-shelley-*-win64.zip" -DestinationPath .
-          Expand-Archive -Force -Path "cardano-wallet-shelley-*-deployments.zip" -DestinationPath shelley_deployments
+          Expand-Archive -Force -Path "cardano-wallet-jormungandr-20*-win64.zip" -DestinationPath .
+          Expand-Archive -Force -Path "cardano-wallet-20*-win64.zip" -DestinationPath .
```

Comment:
20?

Reply:
globbing pattern to cover all versions?

Reply:
`cardano-wallet` is a prefix of `cardano-wallet-jormungandr`. So if I don't put a part of the version in there, the glob will match too much, and powershell implodes.